Makefile

```diff
@@ -2,12 +2,24 @@ ifndef CCPROOT
 	export CCPROOT=$(GOPATH)/src/github.com/crunchydata/crunchy-containers
 endif
 
+.PHONY:	default versiontest
+
+default:	all
```

Comment:
Why not just move the 'all' target up to the top? `make` assumes the 'first' target is the 'default'.

Reply:
@abrightwell Good suggestion.  I have made and pushed that change.